client/src/components/navbar/navbarForm.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Field, reduxForm } from 'redux-form'
+import { RaisedButton, TextField } from 'material-ui'
+import './style.css'
+
+const renderTextField = ({ input }) => (
+  <div className="navbarSearch">
+    <TextField hintText="Search" {...input} />
+  </div>
+)
+
+const SearchBar = props => {
+  const { handleSubmit } = props
+  return (
+    <form className="navbarSearchForm" onSubmit={handleSubmit}>
+      <Field name="searchKey" component={renderTextField} />
+      <RaisedButton className="navbarSubmit" type="submit" label="Search" />
```

Comment:
should this onSubmit prop have the same name as the prop name in the component itself (handleSubmit)?